modules/determining-upgrade-viability-conditiontype.adoc

```diff
@@ -6,9 +6,9 @@
 [id="Understanding_clusteroperator_conditiontypes_{context}"]
 = Understanding cluster Operator condition types
 
-The status of cluster Operators includes their condition type, informing you of the current state of your Operator's health. The following definitions cover a list of some common ClusterOperator condition types. Operators that have additional condition types and use Operator-specific language have been omitted. 
+Similar to cluster Version condition types, the condition type of cluster Operators informs you of the current state of your Operator's health. The following definitions cover a list of some common cluster Operator condition types. Operators that have additional condition types and use Operator-specific language have been omitted. 
```

Comment:
From the context it seems that `cluster Version` should be `ClusterVersion` and `cluster Operator` should be `ClusterOperator`? The capitalized names are usually understood to be kube resources: we have `ClusterVersion`s and `ClusterOperator`s. OCP also has `Operator`s but these are not "cluster operators", they represent OLM-managed operators - they are basically "addons", not "cluster operators".

As a reader, I think we should either use these names directly, or find some creative ways to prevent confusion. I think it would be fine to speak about "cluster operators" (all smallcaps). I don't know what could be a good variant for `ClusterVersion`.

Reply:
ofc feel free to ignore me if this is an established terminology elsewhere in the docs

Reply:
We expect users to read these condition types in the webconsole or in the OC CLI. At this point the goal is not help users discover these condition types but when they come across one then they should refer the docs to understand what it means.  

The only change I will suggest would be to replace `Similar to cluster Version condition types` to `Similar to cluster Version operator condition types` to communicate that we are talking about the cluster version operator and not ClusterVersion resource.

Reply:
okay, but "cluster Version" is still making my eyes bleed a little ;)

Reply:
How about **cluster version**? It seems like the IBM style guide would recommend that. @skopacz1 what do you think?
<img width="433" alt="Screenshot 2023-02-20 at 11 14 37 AM" src="https://user-images.githubusercontent.com/114947244/220156371-cc52b314-ac42-4555-8f6b-f1492c2fdc39.png">

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#operator-capitalization cleared this for me :) Operator is always capitalized, but not version. I'll update accordingly :)

Reply:
Sorry for the late response but yeah, that second link seems to provide some better context - capitalize Operator, and if you're following Lala's example to say "cluster Version operator", it should be "Cluster Version Operator" (the bullet list of examples from your link actually has Cluster Version Operator there).

I'm not sure about the IBM style guide reference, but at first glance it seems to be telling people to not create terms out of nowhere and then capitalize a letter in the middle? Like if some term already exists as two words, don't combine them and capitalize the beggining of each word. Could be wrong here